stablehlo/tests/stablehlo_legalize_quantized_op_to_qdq.mlir

```diff
@@ -40,27 +40,6 @@ func.func @ops_per_axis_quantization(
 // CHECK-NEXT: stablehlo.uniform_dequantize %arg0
 // CHECK-NEXT: stablehlo.uniform_quantize %arg0
 
-// -----
-// %arg1 can be a per-axis Quantized
```

Comment:
These tests are excluded as they would fail without having https://github.com/openxla/stablehlo/pull/2462. Both the tests will be added in that PR.